Patch to allow assembling with binutils 2.20. It worked without this patch in 2.19.1
diff --git a/board/mv_feroceon/USP/jump.S b/board/mv_feroceon/USP/jump.S
index 4b6b9ee..58037fc 100644
--- a/board/mv_feroceon/USP/jump.S
+++ b/board/mv_feroceon/USP/jump.S
@@ -29,9 +29,11 @@ disclaimer.
 #include "sys/mvCpuIfRegs.h"
 
 
-jumpStart:
 
 .section	".reset_vector_sect",#alloc, #execinstr
+
+jumpStart:
+
 #if defined(MV_88F6082) || defined(MV_88F6183) || defined(DB_88F5181_OLD) || defined(DB_FPGA) || \
 	defined(MV88F6281) || defined(MV88F6192) || defined(MV88F6180)  || defined(MV_88F6183L) || \
     defined(MV88F6190)
